web/client/src/App.js

```diff
@@ -13,46 +13,53 @@ import "./styles/app.css";
  * @return {MaterialUI} themed app
  */
 function App() {
-  const [token, setToken] = useState("");
+  const [response, setResponse] = useState(null);
   const [responseVisable, setResponseVisable] = useState(false);
+
```

Comment:
Can we make responseVisable to tokenResponseVisable or something like that? I feel as though just response is too vague.

Reply:
Just changed it